tests/bayesian/dropconnect_test.py

```diff
@@ -36,36 +24,18 @@ def forward(self, x):
 
 
 @pytest.mark.parametrize("inplace", (True, False))
-@pytest.mark.parametrize("layers", (['Linear'], ['Linear', 'Conv2d']))
+@pytest.mark.parametrize("layers", (['Linear'], ['Linear', 'Conv2d'], ['Conv2d']))
```

Comment:
I remember that I didnt test only `conv2d` because it wasn't working very well if we change all the conv2ds in the model if I remember correctly the result of experiments. Thats why I stick to the one after linear layer. How it os changed now? have you tested it If the active learning working properly?

Reply:
I did not test that, but I think we should test that dropconnect works on this config too right?

Reply:
yeah so I remember I tested it before and it wasn't working. we should do it again